angular/demo/src/app/samples/slider/vertical.route.ts

```diff
@@ -6,15 +6,14 @@ import {FormControl, FormsModule, ReactiveFormsModule} from '@angular/forms';
 	standalone: true,
 	imports: [AgnosUIAngularModule, ReactiveFormsModule, FormsModule],
 	template: `
-		<h2>Vertical slider</h2>
 		<div class="d-flex" style="height: 350px">
 			<div class="col-6" style="height: 300px">
-				<au-component auSlider [auMin]="0" [auMax]="100" [auStepSize]="1" [auVertical]="true" [formControl]="sliderControl"></au-component>
-				Form control values: {{ sliderControl.value?.join(', ') }}
+				<div auSlider [auMin]="0" [auMax]="100" [auStepSize]="1" [auVertical]="true" [formControl]="sliderControl" [auClassName]="'my-0'"></div>
```

Comment:
Very very minor... In other examples, we tend to set the string inputs in the format ```auClassName="my-0"``` instead of ```[auClassName]="'my-0'"```

Reply:
Good catch ! Thanks !